package.json

```diff
@@ -16,10 +16,10 @@
     "@rollup/plugin-typescript": "^8.2.1",
     "@tsconfig/svelte": "^1.0.10",
     "@types/node": "^14.14.37",
-    "node-sass": "^6.0.0",
     "obsidian": "^0.12.0",
     "rollup": "^2.32.1",
     "rollup-plugin-svelte": "^7.1.0",
+    "sass": "^1.34.0",
```

Comment:
`node-sass` is deprecated and my system was complaining about it, so I swapped it out with `sass`